BUILD.md

```diff
@@ -233,7 +233,7 @@ command:
 
 ```bash
 sudo dnf install git cmake libxcb-devel libX11-devel xcb-util-keysyms-devel \
-        libXrandr-devel wayland-devel zlib-devel lz4-devel libzstd-devel
+        libXrandr-devel wayland-devel zlib-devel lz4-devel libzstd-devel clang
```

Comment:
This is for `clang-format`? Poking around* it looks like the packaging on Fedora changed at version 32, meaning this wouldn't pull in `clang-format` for those versions. Are you on Fedora <= 31?

If the situation is as I am surmising, perhaps the dependency should be on the Fedora `git-clang-format` package which should pull in `clang-format` on all recent Fedora releases, or on `clang-tools-extra` which bundles it on versions from 32 onwards (i.e. including the current release).

Not running Fedora here so unable to test locally.

\* https://bugzilla.redhat.com/show_bug.cgi?id=1815913
   https://stackoverflow.com/a/51719891/506073